### PR TITLE
feat: Add Discover name to column mappings

### PR DESCRIPTION
**Pull request by lynnagara**
This will be needed in order to start using the Discover dataset so we can look up the column names

### CODE REVIEW

File: src/sentry/eventstore/base.py [hunk 1]
```diff
@@ -6,76 +6,107 @@
 from sentry import nodestore
 from sentry.utils.services import Service
 
-Column = namedtuple("Column", "event_name transaction_name alias")
+Column = namedtuple("Column", "event_name transaction_name discover_name alias")
```
Comment (fpacifici) [inline src/sentry/eventstore/base.py]: Did you check none is accessing this tuple positionally ? Otherwise all those places would return the wrong result

Comment (lynnagara) [reply src/sentry/eventstore/base.py]: Yeah. Glad we changed this to namedtuple earlier.

File: src/sentry/eventstore/base.py [hunk 2]
```diff
@@ -6,76 +6,107 @@
...
+    ISSUE = Column("issue", None, "group_id", "issue.id")
+    PROJECT_ID = Column("project_id", "project_id", "project_id", "project.id")
+    TIMESTAMP = Column("timestamp", "finish_ts", "timestamp", "timestamp")
+    TIME = Column("time", "bucketed_end", "time", "time")
+    CULPRIT = Column("culprit", None, "culprit", "culprit")
+    LOCATION = Column("location", None, "location", "location")
+    MESSAGE = Column("message", "transaction_name", "message", "message")
+    PLATFORM = Column("platform", "platform", "platform", "platform.name")
+    ENVIRONMENT = Column("environment", "environment", "environment", "environment")
+    RELEASE = Column("tags[sentry:release]", "release", "sentry:release", "release")
```
Comment (markstory) [inline src/sentry/eventstore/base.py]: Should this name be `release` for discover?

Comment (lynnagara) [reply src/sentry/eventstore/base.py]: Yeah, updated now

**Review by fpacifici — APPROVED** [review]

**Review by markstory — APPROVED** [review]

**Pull request merged**